### PR TITLE
[typo*] 

### PR DESCRIPTION
**Pull request by cryptomolot**
hi!:raising_hand: 

1.deleted links - they don't work, couldn't find good ones
2.other commits are typo fixes

thanks a lot and have a good day :star2:

### CODE REVIEW

File: developer-docs-site/docs/guides/building-from-source.md [hunk 1]
```diff
@@ -106,7 +106,6 @@ If the script above does not work for you, you can install these manually, but i
 1. [Rust](https://www.rust-lang.org/tools/install).
 1. [CMake](https://cmake.org/download/).
 1. [LLVM](https://releases.llvm.org/).
-1. [libssl-dev](https://packages.ubuntu.com/bionic/libssl-dev) and [libclang-dev](https://packages.ubuntu.com/bionic/libclang-dev)
```
Comment (gregnazario) [inline developer-docs-site/docs/guides/building-from-source.md:109]: This I believe is still required

File: developer-docs-site/docs/guides/building-from-source.md [hunk 2]
```diff
@@ -106,7 +106,6 @@ If the script above does not work for you, you can install these manually, but i
 1. [Rust](https://www.rust-lang.org/tools/install).
 1. [CMake](https://cmake.org/download/).
 1. [LLVM](https://releases.llvm.org/).
```
Comment (davidiw) [inline developer-docs-site/docs/guides/building-from-source.md:108]:
```suggestion
1. [LLVM](https://releases.llvm.org/).
1. [libssl-dev](https://packages.ubuntu.com/jammy/libssl-dev) and [libclang-dev](https://packages.ubuntu.com/jammy/libclang-dev)

```

**Review by davidiw — APPROVED** [review]

**Comment by davidiw** [comment]
Thanks as usual

File: developer-docs-site/docs/guides/building-from-source.md (hunk 1 above)
Comment (davidiw) [reply developer-docs-site/docs/guides/building-from-source.md:109]: not sure why we used bionic... but using jammy which is a lts release

**Pull request merged**